libsolidity/ast/ASTJsonExporter.cpp

```diff
@@ -297,6 +297,15 @@ bool ASTJsonExporter::visit(ContractDefinition const& _node)
 	if (!_node.annotation().linearizedBaseContracts.empty())
 		attributes.emplace_back("linearizedBaseContracts", getContainerIds(_node.annotation().linearizedBaseContracts));
 
+	if (!_node.annotation().internalFunctionIDs.empty())
+	{
+		Json::Value internalFunctionIDs(Json::objectValue);
+		for (auto const& [functionDefinition, internalFunctionID]: _node.annotation().internalFunctionIDs)
+			if (internalFunctionID.set())
```

Comment:
Removing the set once would allow us to get rid of this - the map would only contain entries for functions that got an internal ID.